release-content/0.14/release-notes/11426_Computed_States__Sub_States.md

```diff
@@ -96,7 +96,7 @@ App::new()
 
 Just like any other state, computed states and substates work with all of the tools you're used to:
 the `State` and `NextState` resources, `OnEnter`, `OnExit` and `OnTransition` schedules and the `in_state` run condition.
-Make sure to visit [both](https://github.com/bevyengine/bevy/blob/main/examples/state/computed_states.rs) [examples](https://github.com/bevyengine/bevy/blob/main/examples/state/sub_states.rs) for more information!
+Make sure to visit [both](https://github.com/bevyengine/bevy/tree/v0.14.0/examples/state/computed_states.rs) [examples](https://github.com/bevyengine/bevy/tree/v0.14.0/examples/state/sub_states.rs) for more information!
```

Comment:
I believe the right tag is `release-0.14.0` unless `v0.14.0` is going to be created